reporter/pubsub/pubsub.go

```diff
@@ -0,0 +1,109 @@
+package pubsub
+
+import (
+	"cloud.google.com/go/pubsub"
+	"context"
+	"encoding/json"
+	"github.com/openzipkin/zipkin-go/model"
+	"github.com/openzipkin/zipkin-go/reporter"
+	"log"
+	"os"
+)
+
+const defaultPubSubTopic = "opentracing"
```

Comment:
I don't think this should be the name of the for the default topic. This is the zipkin library so I don't think there is a need to mention `opentracing`.

Reply:
Fixed